packages/filebrowser/src/crumbs.ts

```diff
@@ -72,9 +67,14 @@ export class BreadCrumbs extends Widget {
     this._trans = this.translator.load('jupyterlab');
     this._model = options.model;
     this._fullPath = options.fullPath || false;
+    this._minimumLeftItems = options.minimumLeftItems || 0;
+    this._minimumRightItems = options.minimumRightItems || 2;
```

Comment:
In the second line with `||` if someone passes 0 they would get `2`. I think we should use `??` here.
```suggestion
    this._minimumLeftItems = options.minimumLeftItems ?? 0;
    this._minimumRightItems = options.minimumRightItems ?? 2;
```